lib/internal/util.js

```diff
@@ -356,16 +356,17 @@ function isInsideNodeModules() {
 
   // Iterate over all stack frames and look for the first one not coming
   // from inside Node.js itself:
-  for (const frame of stack) {
-    const filename = frame.getFileName();
-    // If a filename does not start with / or contain \,
-    // it's likely from Node.js core.
-    if (!/^\/|\\/.test(filename))
-      continue;
-    return kNodeModulesRE.test(filename);
+  if (Array.isArray(stack)) {
```

Comment:
Maybe just invert the conditional to make here?

Reply:
Yeah, I just wanted to keep the semantics exactly the same as before so `return false;` after everything, even if it's strictly-speaking unreachable from the for loop.